### DISCUSSION TITLE
6 pinned posts limit on Github profile?

### DISCUSSION THREAD

**Discussion by rchak007** [community/community#58431] [Repositories]
### Select Topic Area

Product Feedback

### Body

i want to know is 6 the limit to pin repositories? Because i am unable to pin more than 6.

**Comment by airtower-luna** [comment]
Yep, the menu explicitly says so ("up to six"):
![grafik](https://github.com/community/community/assets/5814919/2c7e8076-1647-442c-bff8-bf25c91f38f2)

**Reply by rchak007** [reply]
yea but why should it be limited and not us choose whatever we want. Anyway Github is amazing!
